etsin_finder/frontend/js/components/qvain/utils/fileHierarchy.js

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Utility functions for handling the file hierarchy structures. Basically helpers for handling deeply hierarchical objects.